web/tests/login_with_email_tests.ts

```diff
@@ -0,0 +1,94 @@
+import * as defaultPage from "../pageObjects/default_page"
+import * as loginPage from '../pageObjects/login_page'
+import * as userCredentials from '../utils/user_credentials'
+import { expect } from "chai"
+import { validationFieldMessages } from '../utils/validation_field_messages'
+
+const incorectPass = 'ecret123'
+const notRegisteredEmail = 'no@email.com'
+const invalidEmails = ['invalidEmail.com', 'invalid@email', '@invalidEmail.com', '!Tº••…¬¬∫√%##%&~“π¬ª¶•.com', 'invalid.@email@invalidEmail.com']
+const shortPass = '1234'
+const longPass = Math.random().toString(16).repeat(10)
+const longEmail = Math.random().toString(16).repeat(18) + '@a.a'
+//--Maybe need to move to other place--//
+const registerPageHeader = 'Register'
+const forgotPassHeader = 'Reset your password'
+
+
+describe('Login with email tests', () => {
+
+    before(async function () {
+        await defaultPage.openSigInPageURL()
+
+    })
+
+    beforeEach(async function () {
+        await browser.refresh()
+
+    })
+
+    //--Need test for succesful login--//
+
+    it('Inccorect Email validation message', async () => {
```

Comment:
test name and test assertion meaning is different. Suggest to change the test name. e.g. similar to `Incorrect Email message while logging in with not registered email`